app/assets/stylesheets/general/_base.sass

```diff
@@ -103,3 +103,7 @@ a.underline
 
 .icon-space-l-half
   margin-left: 0.25em
+
+.api-section-top-margin
+  margin-top: 12%
+
```

Comment:
I've put this line other places and wasn't able to find a place where it took effect.